src/components/search/SearchError.jsx

```diff
@@ -3,9 +3,8 @@ import { Alert } from '@openedx/paragon';
 import { Warning } from '@openedx/paragon/icons';
 import { getSearchErrorMessage } from '../utils/search';
 
-const SearchError = ({ title }) => {
+const SearchError = ({ title = 'content' }) => {
```

Comment:
With the pivot to `SearchUnavailableAlert`, are you using `SearchError` in any net-new spots? If not, I think we can leave this component untouched, and still require the `title` prop.